timeless/reservations/views.py

```diff
@@ -78,6 +80,25 @@ def list_reservations(reservations):
         "restaurants/tables/list.html", reservations=reservations
     )
 
+@bp.route("/retrieve", methods=("GET", "POST"))
```

Comment:
@carlosmiranda @emilianodellacasa Here may be a confusion. We have here mix of a lot end-points. We already use `/api` prefix for API's
here is an example - https://github.com/timelesslounge/timelessis/blob/master/timeless/__init__.py#L94